backend/src/controllers/invoices.js

```diff
@@ -0,0 +1,26 @@
+const useController = require("./useController/index.js");
+const model = require("../models/invoices");
+const what = { single: "invoice", plural: "invoices" };
+const controller = useController(model, what, "contracts");
+
+/**
+ * Gets all contract invoices for a specific contract.
+ *
+ * @param   {FastifyRequest} request FastifyRequest is an instance of the standard http or http2 request objects.
+ * @param   {FastifyReply}   reply   FastifyReply is an instance of the standard http or http2 reply types.
+ * @returns {object}
+ */
+controller.getAll = async (request, reply) => {
```

Comment:
See my comment in the contracts controller same reason. Not sure why you overwrote this method, don't see any thing different.

Reply:
I can change its name too if that's what we decide on. I had to create this function because the base getAll() doesn't take an id but here we need to use the contract id in the model function.